### PR TITLE
Parameterize team name for UI

### PR DESCRIPTION
**Pull request by dbernard**
Add `TEAM_NAME` as a field in the config file to be used as the title
for the page.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% extends "bootstrap/base.html" %}
-{% block title %}HumanGeo Extra Life{% endblock %}
+{% block title %}{{ team_name }}{% endblock %}
```
Comment (arusahni) [inline templates/index.html]: Should we add a suffix here? i.e. `{{ team_name }} Extra Life`?

Comment (dbernard) [reply templates/index.html]: I can see both sides of this, but my thought was that I'd prefer to leave the slot flexible.

Comment (arusahni) [reply templates/index.html]: Alright. In that case, should we rename the parameter to `page_title`?

Comment (dbernard) [reply templates/index.html]: Yeah I can agree to that. If it was used in additional places as a team name it'd be different, but as it stands `page_title` is more descriptive

**Comment by oxhacks** [comment]
👍 I'm cool with this, but I'll give @arusahni the final word on merging it since he had the assignment.

**Comment by arusahni** [comment]
Just waiting for `page_title` to mosey its way in.

**Comment by dbernard** [comment]
Updated to `page_title`.

File: config/config.ini.example [hunk 2]
```diff
@@ -1,6 +1,7 @@
 [APP]
 CLIENT_ID=your_twitch_client_id
 TEAM_ID=your_extra_life_team_id
+TEAM_NAME=your_team_name
```
Comment (arusahni) [inline config/config.ini.example]: Stray `TEAM_NAME`.

**Comment by arusahni** [comment]
Looks good!

**Pull request merged**